src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -450,9 +475,13 @@ public ContainerClientRegistration registerNode(String zaleniumContainerName, UR
         for (Pod pod : list.getItems()) {
 
             if (podIpAddress.equals(pod.getStatus().getPodIP())) {
+                if ("Running".equals(pod.getStatus().getPhase())) {
```

Comment:
Would it be better to do a lower case comparison? In case one day the API response changes?

Reply:
Also, it'd be nice to keep the indentation.

Reply:
Ok